### PR TITLE
Migrate two newsletter from to the new backend

### PR DESCRIPTION
**Pull request by iAdramelk**
New form urls:

- https://teleport-docs-next-8n09omwxl-gravitational.vercel.app/teleport-demo/ (page footer)
- https://teleport-docs-next-8n09omwxl-gravitational.vercel.app/newsletter/
- https://teleport-docs-next-8n09omwxl-gravitational.vercel.app/kubecon-happy-hour/

### CODE REVIEW

File: components/Newsletter/NewsletterEmailSubscribe.tsx [hunk 1]
```diff
@@ -21,85 +83,13 @@ export function NewsletterEmailSubscribe({ ...props }: FlexProps) {
       width="100%"
       minHeight="44px"
       minWidth={["auto", "304px"]}
-      bg={formLoaded ? "transparent" : "lightest-gray"}
       transition={transition([["backgroundColor", "interaction"]])}
```
Comment (Alqanar) [inline components/Newsletter/NewsletterEmailSubscribe.tsx:86]: I see that you deleted the property `bg`. In that case I think we should delete `transition` property also

File: components/Newsletter/useNewsletter.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+      setSubmitting(true);
+
+      try {
+        await onFormSubmit({ ...initialValues, Email: value });
+      } finally {
+        setSubmitting(false);
+      }
+
+      return false;
```
Comment (Alqanar) [inline components/Newsletter/useNewsletter.ts:44]: Should we really return `false` here?

**Review by Alqanar — COMMENTED** [review]
LGTM! But I left some comments. It would be great if you could take a look at them.

**Comment by C-STYR** [comment]
@iAdramelk these changes are also going to break Registration & Registration Form components, which are used for the KubeCon landing page and another about to be released for marketing.  I'm looking at how to update them now based on this PR.

**Comment by iAdramelk** [comment]
> @iAdramelk these changes are also going to break Registration & Registration Form components, which are used for the KubeCon landing page and another about to be released for marketing. I'm looking at how to update them now based on this PR.

@C-STYR Missed registration page. Will migrate it too.

File: components/EmailSubscribe/EmailSubscribe.tsx [hunk 3]
```diff
@@ -1,96 +1,135 @@
...
+          cursor="pointer"
+          flex="0 0 auto"
+          css={css({
+            "&:focus, &:hover": {
+              outline: "none",
+              bg: "light-gray",
+            },
+            "&::disabled": {
+              opacity: 0.5,
+              pointerEvents: "none",
```
[the remark this answers is not in the document]
Comment (iAdramelk) [reply components/EmailSubscribe/EmailSubscribe.tsx:79]: Checked, Pointer Events disables cursor.

**Pull request merged**